config/envoyconfig/bootstrap_test.go

```diff
@@ -43,6 +43,10 @@ func TestBuilder_BuildBootstrapLayeredRuntime(t *testing.T) {
 			"staticLayer": {
 				"overload": {
 					"global_downstream_max_connections": 50000
+				},
+				"re2": {
+					"error_level": 1048576,
+					"warn_level": 1024
```

Comment:
These should be nested under the "max_program_size" key, right?

Reply:
yes